### ISSUE TITLE
Broken pipe when processing large dataset

### ISSUE THREAD

**Issue by abc123yuanrui** [MVIG-SJTU/AlphaPose#682]
![image](https://user-images.githubusercontent.com/37522312/91982045-089dd100-ed6d-11ea-96b6-a86c7e6a3668.png)

**Comment by Fang-Haoshu** [comment]
hi, it seems the pytorch is broken. You may use smaller queue size by --qsize 128 to see if it helps

**Comment by abc123yuanrui** [comment]
> hi, it seems the pytorch is broken. You may use smaller queue size by --qsize 128 to see if it helps

I've tried to write the result as separate JSON files instead of combining them as an queue (made some changes to writer and pPose_nms). However, similar error still popped out. I personally think this might caused by lack of capability from cloud computing platform but not necessarily sure about that. Anyways, I chunked the dataset into some subset in the end. Thank you for your response.
